src/lib.rs

```diff
@@ -0,0 +1,224 @@
+use std::{
+    sync::Mutex,
+    time::{SystemTime, UNIX_EPOCH},
+};
+
+use discord_sdk::{
+    activity::{ActivityBuilder, Assets},
+    user::User,
+    wheel::UserState,
+    wheel::Wheel,
+    Discord, DiscordApp, Subscriptions,
+};
+use rrplug::prelude::*;
+use rrplug::wrappers::presence::{GamePresence, GameStateEnum};
+use tokio::runtime::Runtime;
+
+const APP_ID: i64 = 941428101429231617;
```

Comment:
`APP_ID` is id for our application/discord-integration, right?

Value wasn't explained in old code either ^^"

Reply:
yes.
I took from the c++ version

Reply:
I believe @emma-miler owns that discord app